.circleci/config.yml

```diff
@@ -28,36 +28,50 @@ jobs:
           command: | 
             git config --global user.name "Bodywork Machine Learning Ltd"
             git config --global user.email "info@bodyworkml.com"
+      - add_ssh_keys:
```

Comment:
We don't need this now the tests are split?

Reply:
Yes, this PR isn't ready to review yet so I hadn't cleaned up the config.yml yet.